react/__tests__/__snapshots__/productKit.test.js.snap

```diff
@@ -0,0 +1,82 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`<ProductSummary /> component should match the snapshot 1`] = `
+<DocumentFragment>
+  <section
+    class="container-mock"
+  >
+    <div
+      class="t-heading-3 c-muted-1 fw3 mv4 flex items-center justify-center"
+    >
+      <span>
+        Mount your Kit
+      </span>
+    </div>
+    <div
+      class="slider-mock"
+    >
+      <div
+        class="container flex flex-row items-center justify-center b--muted-5 pv3 w-100"
+      >
+        <div
+          class="flex flex-row items-center justify-center c-muted-1"
+        >
+          <div
+            class="item"
+          >
+            <div
+              class="flex flex-row"
+            >
+              <div
+                class="pl4"
+              >
+                <div
+                  class="extension-point-mock"
+                />
+              </div>
+              <div
+                class="flex flex-column items-center w2"
+              />
+            </div>
+          </div>
+        </div>
+        <span
+          class="c-muted-1"
+        >
+          <svg
+            class="undefined-oritentation-mock w2 IconEquals"
```

Comment:
You know why this orientation is going undefined? Maybe a missing prop in the component.

Reply:
good observation, the icon mock is receving a prop that is never used, I should remove that.